.travis.yml

```diff
@@ -17,7 +17,6 @@ addons:
 
 cache:
   directories:
-  - node_modules
```

Comment:
He's trying to figure out why something isn't working, and needed to clear the cache. It should be temporary.